builders/get-started/networks/moonbeam-dev.md

```diff
@@ -34,6 +34,23 @@ Using Docker enables you to spin up a node in a matter of seconds. Once you have
 
     The tail end of the console log should look like this:
 
+    <!-- termynal -->
+    ```bash
+    moonbeam@ubuntu-jammy$ docker pull moonbeamfoundation/moonbeam:v0.34.1
+
+    v0.34.1: Pulling from moonbeamfoundation/moonbeam
+    1f7ce2fa46ab: Pull complete
+    1208a88c940c: Pull complete
+    55a3907f35d5: Pull complete
+    7f83070ed01f: Pull complete
+    120180f27e1b: Pull complete
+    661ce21cf677: Pull complete
+    450f087713bc: Pull complete
+    Digest: sha256:b84255b2f0b7defec395f38ed04e55bb8ca1e912e958ce42e0cec84094ef4b48
+    Status: Downloaded newer image for moonbeamfoundation/moonbeam:v0.34.1
+    docker.io/moonbeamfoundation/moonbeam:v0.34.1
+    ```
+
     ![Docker - imaged pulled](/images/builders/get-started/networks/moonbeam-dev/moonbeam-dev-1.png)
```

Comment:
I think its unnecessary to keep both the image and the log output here I would pick one